app/models/form/sales/pages/buyer_live_in_value_check.rb

```diff
@@ -0,0 +1,21 @@
+class Form::Sales::Pages::BuyerLiveInValueCheck < Form::Sales::Pages::Person
+  def initialize(id, hsh, subsection, person_index:)
+    super
+    @depends_on = [
+      {
+        "buyer#{person_index}_not_livein?" => true,
```

Comment:
this is so readable I am sooo confident what this method means I'm not going to bother scrolling down. 🎉

Reply:
lol. on reflection, this method didn't actually do what I thought it was going to when I wrote this 😬